plugins/repository-s3/src/internalClusterTest/java/org/opensearch/repositories/s3/S3BlobStoreRepositoryTests.java

```diff
@@ -72,15 +72,12 @@
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST)
 public class S3BlobStoreRepositoryTests extends OpenSearchMockAPIBasedRepositoryIntegTestCase {
 
-    private String region;
+    private final String region = "test-region";
```

Comment:
@dblock for the existing tests, do you think it is a good idea to always scope to a valid region?

Reply:
There's no such a thing as a "valid" region. Regions are added after code ships, so a future region will also be a valid region. It only matters in integration though when code tries to write to an actual region.